shibata added English-version file of the difference after February 10 (#110)
fujitake/vantiq-related: vantiq-cloud-infra-operations/terraform_aws/readme_en.md

- module `eks`  
  - Create an EKS for Public access point   
  - `managed_node_group_config`: The config of a Managed Node Group. One Managed Node Group for each key object (e.g. VNATIQ).  

- module `keycloak-db`  
  - Create a DB instance for the Private endpoint (created in a single node configuration in one of the AZs in the Private Subnet).  
  - `db_name`: Name of DB      
  - `db_username`: User name of DB      
  - `db_password`: Password for the above DB user    
    **Already mentioned in "password1234", so be sure to change it.**  
  - `db_instance_class`: Instance size of the DB instance  
  - `db_storage_size`: Desk size of the DB instance  
  - `db_storage_type`: Disk type of the DB instance  
  - `postgres_engine_version`: The version of PostgreSQL  

- module `opnode`  
  - Create the Bastion host for working.  
  - Set up permissions for access using the SSH key created in the Preparation.  

>   - The Worker Node of the Managed Node Group is only allowed to use SSH from the Bastion host.    

Reviewer: ```suggestion
  - The Worker Node of the Managed Node Group is only accessible via SSH from the Bastion host.    
```
